libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -187,9 +202,17 @@ bool AP_Arming::airspeed_checks(bool report)
             return true;
         }
         for (uint8_t i=0; i<AIRSPEED_MAX_SENSORS; i++) {
-            if (airspeed->enabled(i) && airspeed->use(i) && !airspeed->healthy(i)) {
-                check_failed(ARMING_CHECK_AIRSPEED, report, "Airspeed %d not healthy", i + 1);
-                return false;
+            if (airspeed->enabled(i)) {
+                if (!airspeed->healthy(i)) {
+                    check_failed(ARMING_CHECK_AIRSPEED, report, "Airspeed %d not healthy", i + 1);
+                    return false;
+                } else if (airspeed->get_airspeed() > 7.01f) {
+                    check_failed(ARMING_CHECK_AIRSPEED, report, "Airspeed %d unexpected high airspeed", i + 1);
```

Comment:
Should print out what the current airspeed is versus threshold so the user can assess it better, particularly if it's transient.

Reply:
Done. Please check.